src/libOpenImageIO/CMakeLists.txt

```diff
@@ -4,6 +4,14 @@ endif ()
 file (TO_NATIVE_PATH "${PLUGIN_SEARCH_PATH}" PLUGIN_SEARCH_PATH_NATIVE)
 configure_file (imageio_pvt.h.in "${CMAKE_BINARY_DIR}/include/imageio_pvt.h" @ONLY)
 
+# install pkgconfig file
+IF ( NOT WIN32 )
+   configure_file(OpenImageIO.pc.in "${CMAKE_BINARY_DIR}/include/OpenImageIO.pc" @ONLY) 
```

Comment:
Is there any reason why this should get created in the include directory?

Can I just say `${CMAKE_BINARY_DIR}/OpenImageIO.pc` ??

Reply:
i'd say yes.  i just tried it w/o the "include" and it still installs in lib/pkgconfig correctly.

i modeled it after the way imageio_pvt.h.in was done btw.

Reply:
Right, but imageio_pvt.h is actually an include file. Let's move it as I suggested, just to clarify that it's unrelated to the header files. 

And similarly, I'm not sure the .pc.in file ought to live in src/libOpenImageIO, because it is in fact not at all related to that library, right? It's about the whole package overall. I wonder if build-scripts is the right directory to put it?

Reply:
that sounds good.  i'll make that change.

Reply:
if OpenImageIO.pc.in moves to build-scripts i'm assuming you want to move the changes to src/libOpenImageIO/CMakeLists.txt to build-scripts as well?  if so would something like the build_openexr.bash script work for this?  something like build-pkgconfig.bash?

Reply:
No, sorry, maybe I was not specific enough. It's simpler than that.

This patch was (right up to a few minutes ago) consisting of two things:
(1) a new template file, OpenImageIO.pc.in, which you first put in src/include.
(2) a few modified lines right in the main CMakeLists.txt, which generates OpenImageIO.pc from the `in` template file and puts it temporarily in build/include.

ALL I'm saying is: 
(a) for item 1, put the new .pc.in file in src/build-scripts instead of src/include.
(b) leave the logic in the main CMakeLists.txt, but have the temporary spot where your write the pc file be just in build rather than build/include.

Reply:
the problem with this is if i move OpenImageIO.pc.in into src/build-scripts and the logic is in src/include/CMakeLists.txt i'd have to do:

`configure_file(../build-scripts/OpenImageIO.pc.in, ...`

and that's kinda smelly.  are you fine with that?

or am i missing something?

Reply:
The logic should be in the main CMakeLists.txt file, it should never have been in src/include/CMakeLists.txt. I understand why you put it there -- it was the only other instance of using cmake to write a file based on a template. But it doesn't belong there because it's unrelated to include files.

Reply:
ah... that makes sense.  ok.  will have something shortly.

Reply:
I appreciate your patience. I hate to make you jump through these extra hoops. Just trying to situate it in the most logical place and then we can forget about it all for a long long time without it keeping us awake at night. :-)

Reply:
oh no problem at all. it didn't make much sense being where it was in the first place.